server/utils/viewUtils.ts

```diff
@@ -13,6 +13,14 @@ export default class ViewUtils {
     return escape(val).val
   }
 
+  static nl2br(val: string): string {
+    // handle things like nunjucks macros (which are 'SafeStrings') that do not need formatting
```

Comment:
I'm not sure I understand this comment. Do you mean "Nunjucks macros" or "the return value of a Nunjucks macro"?

Reply:
will clarify this comment. it's mainly the govuk tag macro. the thing where we pass the macro from the template back to anonymous js functions means this method ends up getting called with non-string arguments. it surprised me too when i saw it.